lib/src/main/java/com/auth0/android/lock/views/LogInFormView.java

```diff
@@ -179,7 +179,14 @@ private void resetDomain() {
     }
 
     private String getUsername() {
-        return currentConnection == null && fallbackToDatabase ? emailInput.getText() : usernameInput.getText();
+        boolean usingDatabase = currentConnection == null && fallbackToDatabase;
+        boolean usingOAuthEnterprise = currentConnection != null && !currentConnection.isActiveFlowEnabled();
+        if (usingDatabase || usingOAuthEnterprise) {
+            return emailInput.getText();
+        } else {
```

Comment:
The previous check was not considering the `activeFlow` value of the connection, so now there are 3 cases:
1. It's a db connection -> use /ro
2. It's a enterprise connection with active flow -> use /ro
3. it's a enterprise connection without active flow -> use /authorize

What I require in the last case is to provide the username/email typed before clicking submit and send as `login_hint` in the /authorize call. (This is the PR).

In this method is where I take that case and chose the right `EditText`, as in the case of (1) and (2) there are 2 steps/screens: the first one shows email and password inputs and if the typed domain matches, it will hide the password and replace the email with a username input.